### DISCUSSION TITLE
Pulling YAML data from another note file

### DISCUSSION THREAD

**Discussion by jwolfe504** [SilentVoid13/Templater#751] [Help]
I have YAML data a note file which I update periodically. I also have a template that create a new note and I'd like to be able to pull the data from the one note file and put it into the new note. Is this possible?

**Comment by Mara-Li** [comment]
hello !
You could use `<% tp.file.include("[[yourfile]]") %>`

Also, the templater in te file will be run, so you can use multiple different file and combine them with that. 
Ah, yes, you can also take a part of the file, with `#header` (as an embed)
